charlesinto/iReporter
# #162579568 Get all intervention Record

#### What does this PR do?
- Implement JWT authentication
- Verify user role
- Get all intervention records
- If the user gets all user records
- If admin get all records


#### Description of Task to be completed?
- implement jwt authentication
- verify user token
- user should be able to post new report

#### How should this be manually tested?
- Clone the repo - `git clone  git repo address`
- Enter into the project directory - `cd repo folder`
- switch to this branch - `PR branch`
-  Start  the frontend.
- Navigate to `endpoint`
- Open `Postman`
- Grab your token from the application tab in Developers tab
- On postman add the token in headers
- - On `postman` Send a `GET` request to `http://localhost:7000/api/v1/interventions`
- You should receive a successful response

#### Any background context you want to provide?


`

#### What are the relevant pivotal tracker stories?
[#162579568 ](https://www.pivotaltracker.com/story/show/162579568 )

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/26615085/49822004-2e569b80-fd7c-11e8-9978-65b604bb293a.png)





#### Questions:
N/A
